src/javascript/app/pages/trade/price.js

```diff
@@ -355,6 +345,7 @@ const Price = (() => {
 
         processForgetProposalOpenContract();
         processForgetProposals().then(() => {
+            $('.price_container').hide();
```

Comment:
it looks strange that it keeps disappearing and reappearing, isn't there any other way than making it display none all the time?

Reply:
@negarn It doesn't really reappear, it just persists from previous contract type. Here I hide all price containers on contract type change, then `display()` shows the necessary ones.